Tests/AutoCheckTools.h

```diff
@@ -13,6 +13,22 @@
 
 #include "MathLib/Point3d.h"
 
+
+namespace std{
```

Comment:
There is no output operator for std::array in the STL. The operator is required for output of the arrays in case of a failing test.
One could move the operator to `autocheck` namespace but it must be visible _before_ the `autochcek.hpp` include...
Having it in the `std` namespace allows for an argument dependent lookup.